packages/api/src/praise/utils/core.ts

```diff
@@ -74,3 +75,37 @@ export const isQuantificationCompleted = (
     quantification.score > 0
   );
 };
+
+export const prepareDiscordClient = async (): Promise<Client> => {
+  const discordClient = new Client({
+    intents: ['GUILDS', 'GUILD_MEMBERS'],
+  });
+  await discordClient.login(process.env.DISCORD_TOKEN);
+  await discordClient.guilds.fetch();
+  const discordGuild = await discordClient.guilds.fetch(
+    process.env.DISCORD_GUILD_ID as string
+  );
+  await discordGuild.members.fetch();
+
+  return discordClient;
+};
+
+export const generateReasonRealized = async (
+  discordClient: Client,
+  sourceId: string,
+  reason: string
+): Promise<string> => {
+  const parsedSourceId = sourceId.match(/DISCORD:[\d]+:([\d]+)/);
```

Comment:
When running a local development setup, API startup gets stuck here because of invalid Discord source Id.